components/automate-load-balancer/habitat/config/automate-host-header-validation.conf

```diff
@@ -0,0 +1,9 @@
+map $http_host $istrusted {
+ 
+    default false;
+
+    localhost true;
```

Comment:
`localhost` and `127.0.0.1` an be used to bypass arbitrary host header check. Let's keep only for FQDN only.